githooks/hooks/trusted.go

```diff
@@ -238,7 +238,21 @@ func (t *ChecksumStore) Summary() string {
 
 // GetChecksumDirectoryGitDir gets the checksum file inside the Git directory.
 func GetChecksumDirectoryGitDir(gitDir string) string {
-	return path.Join(gitDir, ".githooks.checksums")
+	var conf string
+	gitx := git.NewCtxAt(gitDir)
+	scope := git.Traverse
+	conf, set := os.LookupEnv("GITHOOKS_CHECKSUMS_DIR")
+	if !set {
+		conf = gitx.GetConfig(GitCKChecksumsDir, scope)
+	}
+
+	exists, _ := cm.IsPathExisting(path.Join(gitDir, conf))
+	switch {
+	case !strs.IsEmpty(conf) && exists:
```

Comment:
```suggestion
	case !strs.IsEmpty(conf):
```

**suggestion**: You should probably remove the path check, as then the directory gets created, if the setting is set it is used.